app/services/prefabs/prefabs.ts

```diff
@@ -1,20 +1,13 @@
 import { PersistentStatefulService } from 'services/persistent-stateful-service';
-import {
-  ISource,
-  ISourceAddOptions,
-  Source,
-  SourcesService,
-  TPropertiesManager,
-  TSourceType,
-} from 'services/sources';
+import { ISourceAddOptions, SourcesService, TSourceType, Source } from 'services/sources';
 import { ScenesService, TSceneNode, TSceneNodeType } from 'services/scenes';
 import { Inject } from '../../util/injector';
 import { mutation, ServiceHelper } from '../stateful-service';
 import uuid from 'uuid/v4';
 import { TObsValue } from 'components/obs/inputs/ObsInput';
 import Utils from '../utils';
 import Vue from 'vue';
-
+import { HardwareService } from '../hardware';
```

Comment:
Nitpick: Should be a blank line after the last import.  I thought this was a prettier/ts-lint rule but maybe we don't have it enabled.  Also, import should be `import { HardwareService } from 'services/hardware';`